src/scheduler/workers/unpublish/weekly/__tests__/findReminderToSend.test.ts

```diff
@@ -0,0 +1,72 @@
+import { findReminderToSend } from "../findReminderToSend";
+
+const weeklyReminders = {
+  1: "2023-02-08T00:00:00.000Z",
+  2: "2023-02-15T00:00:00.000Z",
+  3: "2023-02-22T00:00:00.000Z",
+  4: "2023-03-01T00:00:00.000Z",
+  5: "2023-03-08T00:00:00.000Z",
+};
+
+const annualReview = {
+  START: "2023-02-01T00:00:00.000Z",
+  POST_ONE_DAY: "2023-01-31T00:00:00.000Z",
+  POST_ONE_WEEK: "2023-01-25T00:00:00.000Z",
+  POST_ONE_MONTH: "2023-01-04T00:00:00.000Z",
+};
+
+const start = new Date(annualReview.START).toISOString().split("T")[0];
+describe(`findReminderToSend when start date is ${start}`, () => {
+  test.each`
```

Comment:
Never seen test.each used before. Looks cool.

Reply:
yep. works really well if you've got lots of repetitive tests and you want to test lots of different scenarios. you can combine describe.each and test.each to form a matrix of tests